src/main/java/org/bptlab/cepta/Actor.java

```diff
@@ -0,0 +1,16 @@
+package org.bptlab.cepta;
+
+public class Actor{
+  public int id;
+  public String name;
+  public Actor(){
+
+  }
+  public Actor(int id, String name){
+    this.id = id;
+    this.name = name;
+  }
+  public String columnString(){
+    return "(id, name)";
+  }
+}
```

Comment:
Not sure if Testing Data should be in Master Branch, this issue should work with the Train Data.

Reply:
I can remove it from the branch :)